### DISCUSSION TITLE
Is it possible to style a rounded table?

### DISCUSSION THREAD

**Discussion by manak1** [chakra-ui/chakra-ui#5368] [Q&A]
I want to style a rounded table like an example on [Chakra UI docs ](https://chakra-ui.com/docs/data-display/table)👇

![スクリーンショット 2022-01-11 14 54 36](https://user-images.githubusercontent.com/32151765/148889030-c96401b0-cc5e-4f7d-a8c8-96d9533a3990.png)

I copied the code on the docs, but it wasn't rounded like an example. 
I also passed a `rounded` props  but it didn't work👇

![スクリーンショット 2022-01-11 15 05 22](https://user-images.githubusercontent.com/32151765/148889961-b44bbf41-cb06-4f58-a537-5ddceff5bfcb.png)

I think the problem is that by default for the table tag `border-collapse` is set to `collapse` 👇

![スクリーンショット 2022-01-11 15 02 43](https://user-images.githubusercontent.com/32151765/148889782-9a586fd9-78ec-4388-9e51-b8a7acc40815.png)

When I changed the `border-collapse` to `separate` border was rounded👇

![スクリーンショット 2022-01-11 15 11 14](https://user-images.githubusercontent.com/32151765/148890611-31e22d89-1c80-46f5-97f9-a97f2c560f60.png)

I tried to override the property by passing `borderCollapse='separate'`. But it doesn't seem like Table component doesn't accept `borderCollapse` as props. Is there any way I can style a rounded table?👀

```
 <Tble
      variant="simple"
      borderRadius="2xl"
      size="lg"
      borderWidth="1px"
      borderCollapse="separate" // this doesn't work
    >
      <TableCaption>Imperial to metric conversion factors</TableCaption>
      <Thead>
        <Tr>
          <Th>example</Th>
          <Th>test</Th>
        </Tr>
      </Thead>
      <Tbody>
        <Tr>
          <Td>inches</Td>
          <Td>millimetres (mm)</Td>
        </Tr>
        <Tr>
          <Td>feet</Td>
          <Td>centimetres (cm)</Td>
        </Tr>
        <Tr>
          <Td>yards</Td>
          <Td>metres (m)</Td>
        </Tr>
      </Tbody>
      <Tfoot>
        <Tr>
          <Th>To convert</Th>
          <Th>into</Th>
        </Tr>
      </Tfoot>
    </Tble>

```

**Comment by brunogayet** [comment]
I have the same problem. Any solution found?

**Comment by NullP0int** [comment]
Got the same problems. Didn't find a solution yet.

**Reply by manak1** [reply]
@brunogayet @NullP0int 

I added the ` style={{ borderCollapse: 'separate' }} ` to the Table Component and it worked.
Maybe this will also solve you guys problem 👀

**Comment by rahabonyani** [comment]
I wrote this and it worked
` <BaseTable
      variant={variant}
      colorScheme="gray"
      mt="8"
      sx={{
        borderCollapse: 'separate',
        borderSpacing: '0 15px',
      }}
    >`

![image](https://user-images.githubusercontent.com/67738966/163675607-afab5905-8d59-4271-ab5f-6d9088a90f7c.png)

**Reply by matbmotta** [reply]
This one helped me a lot thanks!!, but if I want to put border radius at the Tbody on each <Tr></Tr>? How can I do it?

**Reply by Jarrodsz** [reply]
this is really not possible? ouch... should the chakra team look at this before adding new features?

**Comment by dbousamra** [comment]
You can also do this:

```
__css={{
  'tr:last-child td': {
    borderWidth: 0,
  },
}}
```
